### PR TITLE
Static IP address troubleshooting

### PR DESCRIPTION
**Pull request by benjaoming**
This is a subject that's hard to cover, but I feel like we need to add some words about it, since it's been a recent subject of support questions.

### CODE REVIEW

File: docs/manage/_get_support.rst [hunk 1]
```diff
@@ -91,3 +93,29 @@ About IP addresses
...
+.. _ips_static:
+
+Static IP addresses
+-------------------
+
+In order for other clients on your network to find Kolibri on a consistent IP address, this address shouldn't change.
+
+The default behavior of an operating system, no matter if it's Linux/Windows/Mac, will be to receive an IP address from a network authority, i.e. a *DHCP server*. DHCP is a service that's typically run on the local access point or router.
+
+Therefor, you should be cautious that when such a router or access point restarts (for instance during a power cut), it may forget the IP address assigned to your Kolibri device, and thus the IP changes.
```
Comment (radinamatic) [inline docs/manage/_get_support.rst]: Missing _*e*_ in `Therefor`.

**Comment by radinamatic** [comment]
Just a bit confused about the included files: you reference `_troubleshooting.rst`, but all the newly added content is still in the `_get_support.rst` and does not get included once the docs are built... :confused: 

Is one of those two files meant to be deleted?

**Review by radinamatic — CHANGES_REQUESTED** [review]
Check what you are using the `include` directive for!

File: docs/manage/_get_support.rst [hunk 2]
```diff
@@ -1,4 +1,4 @@
-.. _support:
+.. _troubleshooting:
```
Comment (indirectlylit) [inline docs/manage/_get_support.rst:1]: your updates regarding static IPs are not showing up - looks like you may have started refactoring page names but didn't finish?

There is now content that's duplicated between this file (`_get_support.rst`) and the newly-created `_troubleshooting.rst`

**Review by indirectlylit — DISMISSED** [review]

File: docs/manage/_get_support.rst (hunk 2 above)
Comment (indirectlylit) [reply docs/manage/_get_support.rst:1]: whoops sorry - didn't see @radinamatic's review. I'll close mine

**Comment by indirectlylit** [comment]
these updates look great overall.

Made some minor suggestions here: https://github.com/benjaoming/kolibri-docs/pull/1/files

**Comment by benjaoming** [comment]
Sorry about the confusion, yes it was indeed an artifact that should not have been in the PR. It's fixed now.

**Review by radinamatic — APPROVED** [review]

**Pull request merged**